webapp/src/components/tutorial/TutorialHint.tsx

```diff
@@ -24,9 +24,6 @@ export function TutorialHint(props: TutorialHintProps) {
     return <TutorialCallout className="tutorial-hint"
         buttonIcon="lightbulb"
         onClick={onHintClick}>
-            <div className="hint-title">
-                <span>{lf("Answer Key")}</span>
-            </div>
-            {markdown && <MarkedContent className="hint-content" markdown={markdown} unboxSnippets={true} parent={parent} />}
+            {markdown && <MarkedContent markdown={markdown} unboxSnippets={true} parent={parent} />}
```

Comment:
```suggestion
            {markdown && <MarkedContent className="hint-content" markdown={markdown} unboxSnippets={true} parent={parent} />}
```

actually lets leave the class, it has a small fix for width resizing